src/actions/ast.js

```diff
@@ -79,6 +79,20 @@ export function clearSelection() {
   };
 }
 
+function findSymbol(symbols, tokenPos) {
+  const array = Object.keys(symbols).reduce(
```

Comment:
we probably need an explicit order here to prefer more specific types like `member expression`...

Or not if the start/end loc help out